### ISSUE TITLE
Project 1st phase

### ISSUE THREAD

**Issue by mattia93** [mattia93/armadillo#1]
### Requirements:
- Use fashionMNIST as dataset
- Little Hyperparameters tuning (basically only the hidden layer) 
- Loss: binarycrossentropy
- Output activation: softmax
- Early stopping metric: val_loss with patience 10 

### TO-DO
- [x] Create models with different hidden activation functions 
- [x] Attach each one of these models
- [x] Confidence degree of the attach 
- [x] Comparison of metrics before and after attacks
- [x] Single class analysis of the attach
